modules/apps/object/object-api/src/main/java/com/liferay/object/system/util/SystemObjectDefinitionManagementChecker.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.system.util;
+
+import com.liferay.batch.engine.util.BatchEngineThreadLocal;
+import com.liferay.portal.kernel.util.SetUtil;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.util.Collections;
+import java.util.Set;
+
+/**
+ * @author Gabriel Albuquerque
+ */
+public class SystemObjectDefinitionManagementChecker {
+
+	public static boolean isInvokerBundleAllowed() {
+		String bundleNamespace =
+			BatchEngineThreadLocal.getInvokerBundleNamespace();
+
+		for (String allowedInvokerBundleSymbolicName :
+				_allowedInvokerBundleSymbolicNames) {
+
+			if (StringUtil.startsWith(
+					bundleNamespace, allowedInvokerBundleSymbolicName)) {
+
+				return true;
+			}
+		}
+
+		return false;
+	}
+
+	private static final Set<String> _allowedInvokerBundleSymbolicNames =
```

Comment:
Hey guys I think this would fit better next to [these ones](https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/definition/util/ObjectDefinitionUtil.java#L53-L96) to keep all the allow lists in the same place. What do you think?

Reply:
Good suggestion, @4lejandrito! 
@seltong can you move it? Thx

Reply:
Hey guys I believe we cannot do that yet. The ObjectDefinitionUtil class is inside of an internal directory as you can see [here](https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/object/object-service/src/main/java/com/liferay/object/internal/definition/util/ObjectDefinitionUtil.java#L6).
I will send a follow-up moving this `ObjectDefinitionUtil` to the `object-api` and then I will move the code of the `SystemObjectDefinitionManagementChecker` as you suggested.

Does it makes sense?

Reply:
I think it does make sense to move it to api.